### ISSUE TITLE
Linting command broken by dtslint@0.5.6

### ISSUE THREAD

**Issue by ericyhwang** [DefinitelyTyped/DefinitelyTyped#34133]
See https://github.com/Microsoft/dtslint/issues/219 for details. I'm also filing this here, as I'm guessing many people will look here first instead of in dtslint.

The error looks like this:

```
$ npm run lint ____

> definitely-typed@0.0.3 lint .../DefinitelyTyped
> dtslint types "____"

.../DefinitelyTyped/node_modules/.bin/dtslint: line 1: use strict: command not found
.../DefinitelyTyped/node_modules/.bin/dtslint: line 2: syntax error near unexpected token `('
.../DefinitelyTyped/node_modules/.bin/dtslint: line 2: `var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {'
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! definitely-typed@0.0.3 lint: `dtslint types "____"`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the definitely-typed@0.0.3 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

## Workaround

Use the previous version of dtslint.

```
npm install --no-save dtslint@0.5.5
npm run lint $PACKAGE_NAME
```

**Comment by alextes** [comment]
Instead of using the previous version one can also:
```
node node_modules/.bin/dtslint types package-name
```
or prepend the missing `#!/usr/bin/env node` manually to `node_modules/.bin/dtslint`.

**Comment by ericyhwang** [comment]
Fixed by dtslint@0.5.8 - `npm update dtslint` inside DefinitelyTyped should pick up the fix.
